include/deal.II/matrix_free/evaluation_template_factory_internal.h

```diff
@@ -45,6 +45,8 @@ namespace internal
         else if (n_q_points_1d == (3 * degree) / 2 + 1)
           return EvaluatorType::template run<degree, (3 * degree) / 2 + 1>(
             args...);
+        else if (n_q_points_1d == (2 * degree))
+          return EvaluatorType::template run<degree, (2 * degree)>(args...);
```

Comment:
How much larger to the libraries get? I would suggest to limit this to low-ish degrees, say
```suggestion
        else if (n_q_points_1d == (2 * degree) && degree <= 4)
          return EvaluatorType::template run<degree, (2 * degree)>(args...);
```
because otherwise we have too many code paths to compile.

Reply:
@kronbichler I have added the check!